cmd/orphan-finder/main.go

```diff
@@ -235,7 +232,23 @@ func storeParsedLogLine(sa certificateStorage, ca ocspGenerator, logger blog.Log
 	return true, true, typ
 }
 
-func setup(configFile string) (blog.Logger, core.StorageAuthority, core.CertificateAuthority) {
+func generateOCSP(ctx context.Context, ca ocspGenerator, certDER []byte) ([]byte, error) {
+	// generate a fresh OCSP response
+	statusGood := string(core.OCSPStatusGood)
+	zeroInt32 := int32(0)
+	zeroInt64 := int64(0)
+	ocspResponse, err := ca.GenerateOCSP(ctx, &capb.GenerateOCSPRequest{
+		CertDER:   certDER,
+		Status:    &statusGood,
+		Reason:    &zeroInt32,
+		RevokedAt: &zeroInt64,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return ocspResponse.Response, nil
+}
+func setup(configFile string) (blog.Logger, core.StorageAuthority, capb.OCSPGeneratorClient) {
```

Comment:
```suggestion

func setup(configFile string) (blog.Logger, core.StorageAuthority, capb.OCSPGeneratorClient) {
```

Reply:
You're missing a `\n` between `func setup(configFile string) ....` and the `}` closing out the `generateOCSP` scope.